.github/workflows/cd.yml

```
@@ -12,7 +12,29 @@ on:
 
 
 jobs:
+  autoblack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Black
+        run: pip install black
+      - name: Run black --check .
+        run: black --check -S jina/drivers/__init__.py
+      - name: If needed, commit black changes to the pull request
+        if: failure()
+        run: |
+          git checkout $GITHUB_HEAD_REF
+          black -S jina/drivers/__init__.py
+          git config --local user.email "dev-bot@jina.ai"
+          git config --local user.name "Jina Dev Bot"
+          git add . && git commit -m "update code-style ${{env.JINA_VERSION}} due to ${{github.event_name}} on ${{github.repository}}" && git push
```

```suggestion
          git add . && git commit -m "update code-style ${{env.JINA_VERSION}} due to ${{github.event_name}} on ${{github.repository}}" && git push
```

this has two problems:
- recursive triggering CD flow
- commit msg is not styled

---

Fixed. Can you have another look? I took some more inspiration from `copyright.yml`